site/src/main/resources/bl-override-ehcache.xml

```diff
@@ -121,5 +121,92 @@
         maxElementsInMemory="5000"
         eternal="true"
         overflowToDisk="true"/> -->
-        
+    <cache alias="blProducts">
+        <expiry>
+            <ttl unit="seconds">3</ttl>
+        </expiry>
+        <heap>1000</heap>
```

Comment:
not sure if any cache should have heap 1000
Also looking into doc it should be
<heap unit="entries">xxxx</heap>